### PR TITLE
matched func_80240038_7E73D8, func_80240040_7E73E0

### PR DESCRIPTION
**Pull request by Mathyland**
* matched func 80240038_7E73D8

* matched func func_80240040_7E73E0

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀 This PR matches 2 functions (+0.00%) on `us`.

File: src/world/area_mac/machi/7E73A0.c [hunk 1]
```diff
@@ -77,9 +77,13 @@ ApiStatus N(80240030_7E73D0)(Evt* script, s32 isInitialCall) {
     return ApiStatus_DONE2;
 }
 
-INCLUDE_ASM(s32, "world/area_mac/machi/7E73A0", func_80240038_7E73D8);
+s32 func_80240038_7E73D8(void) {
```
Comment (ethteck) [inline src/world/area_mac/machi/7E73A0.c:80]:
```suggestion
ApiStatus func_80240038_7E73D8(Evt* script, s32 isInitialCall) {
```

File: src/world/area_mac/machi/7E73A0.c [hunk 2]
```diff
@@ -77,9 +77,13 @@ ApiStatus N(80240030_7E73D0)(Evt* script, s32 isInitialCall) {
...
+    return 2;
```
Comment (ethteck) [inline src/world/area_mac/machi/7E73A0.c:81]:
```suggestion
    return ApiStatus_DONE2;
```

File: src/world/area_mac/machi/7E73A0.c [hunk 3]
```diff
@@ -77,9 +77,13 @@ ApiStatus N(80240030_7E73D0)(Evt* script, s32 isInitialCall) {
...
+}
 
-INCLUDE_ASM(s32, "world/area_mac/machi/7E73A0", func_80240040_7E73E0);
+s32 func_80240040_7E73E0(void) {
```
Comment (ethteck) [inline src/world/area_mac/machi/7E73A0.c:84]:
```suggestion
ApiStatus func_80240040_7E73E0(Evt* script, s32 isInitialCall) {
```

File: src/world/area_mac/machi/7E73A0.c [hunk 4]
```diff
@@ -77,9 +77,13 @@ ApiStatus N(80240030_7E73D0)(Evt* script, s32 isInitialCall) {
...
+    return 0;
```
Comment (ethteck) [inline src/world/area_mac/machi/7E73A0.c:85]:
```suggestion
    return ApiStatus_BLOCK;
```

**Review by ethteck — CHANGES_REQUESTED** [review]
Here ye go

**Pull request merged**